docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use RadioGroup when you need a clear answer to a binary question that requires form submission."
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+
+  return (
+    <Flex gap={4} direction="column">
+      <RadioGroup legend="Feed preference" id="bestPracticeFeedsDo">
+        <RadioGroup.RadioButton
+          checked={favorite === "grid"}
+          id="grid-do"
+          label="Grid"
+          name="feed-do"
+          onChange={() => setFavorite('grid')}
+          value="grid"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === "list"}
+          id="list-do"
+          label="List"
+          name="feed-do"
+          onChange={() => setFavorite('list')}
+          value="list"
+        />
+      </RadioGroup>
+      <Button color="red" text="Submit"/>
+    </Flex>
+  );
+}
+        `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use a RadioGroup to turn a state on and off with immediate effect on mobile; use [Switch](/switch) instead."
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+
+  return (
+    <Flex gap={4} direction="column">
+      <Flex direction="column" gap={2}>
+        <Text size="400" weight="bold">Auto-renew subscription</Text>
+        <Text size="200">
+          Change will auto-save
+        </Text>
+      </Flex>
+      <RadioGroup direction="row" legend="Auto-renew subscription" legendDisplay="hidden" id="bestPracticeFeedsDont">
+        <RadioGroup.RadioButton
+          checked={false}
+          id="on-dont"
+          label="On"
+          name="feed-dont"
+          onChange={() => setFavorite('on')}
+          value="on"
+        />
+        <RadioGroup.RadioButton
+          checked={false}
+          id="Off-do"
+          label="Off"
+          name="feed-dont"
+          onChange={() => setFavorite('Off')}
+          value="Off"
+        />
+      </RadioGroup>
+      </Flex>
+  );
+}
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Accessibility">
+        <MainSection.Subsection
+          title="Labels"
+          description="Each RadioButton in a RadioGroup should have a label that can be read by screen readers, and that can be clicked or tapped to make it easier for users to select and deselect options. Therefore, make sure to supply the label prop. If that’s not possible, make sure your stand-alone Label has an `htmlFor` prop that matches the `id` of the RadioButton. Test that a RadioButton and label are properly connected by clicking or tapping on the label and confirming that it activates the RadioButton next to it."
```

Comment:
nit: check we are consistent on how we write stand-alone, stand alone, standalone